### ISSUE TITLE
Add flownode instance endpoints' changes

### ISSUE THREAD

**Issue by tmetzke** [camunda/camunda#23746]

**Comment by aleksander-dytko** [comment]
@tmetzke as part of this issue, can we also rename `flownode` to `element`

**Comment by tmetzke** [comment]
No, we should create a dedicated issue for that. This here is only about creating a migration guide and we should keep it to that.
